packages/run-protocol/src/vaultFactory/liquidation.js

```diff
@@ -31,10 +52,16 @@ const liquidate = async (
   burnLosses,
   strategy,
   collateralBrand,
+  penaltyPoolSeat,
+  penaltyRate,
 ) => {
   innerVault.liquidating();
-  const debt = innerVault.getCurrentDebt();
-  const { brand: runBrand } = debt;
+
+  const debtBeforePenalty = innerVault.getCurrentDebt();
+  const penalty = floorMultiplyBy(debtBeforePenalty, penaltyRate);
```

Comment:
The contract usually gets the benefit of rounding:
```suggestion
  const penalty = ceilMultiplyBy(debtBeforePenalty, penaltyRate);
```